### PR TITLE
(PC-10051) Make touchable area when selecting day for Android

### PR DESCRIPTION
**Pull request by antoinewg**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10051

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added a **screenshot** for UI tickets.

## Screenshots

| \*iOS  | \*Android  | 
| -------------------- | ---------------------- | 
| ios | android |

For Android, the touchable area is in green.
For iOS, it's the green area + a "padding" of 8 in each direction due to the `hitSlop`.

### CODE REVIEW

File: src/features/bookOffer/components/Calendar/DayComponent.tsx [hunk 1]
```diff
@@ -19,32 +18,36 @@ interface Props {
   date: DateObject
 }
 
-const hitSlop = { top: 8, bottom: 8, left: 8, right: 8 }
+type VoidFn = () => void
```
Comment (emmajmn) [inline src/features/bookOffer/components/Calendar/DayComponent.tsx:21]: pourquoi tu l'as extrait ? tu aurais pu mettre direct le typage dans la fonction non ?

**Review by emmajmn — COMMENTED** [review]
sinon top 👌🏻

File: src/features/bookOffer/components/Calendar/DayComponent.tsx (hunk 1 above)
Comment (antoinewg) [reply src/features/bookOffer/components/Calendar/DayComponent.tsx:21]: j'ai essayé mais ça marchait pas. quand je sauvegardais, ça mettait `() => void | undefined`. Dans l'autre sens aussi

**Comment by antoinewg** [comment]
Closing because done in https://github.com/pass-culture/pass-culture-app-native/pull/1367

**Pull request closed without merging**